sql/wsrep_schema.cc

```diff
@@ -1247,6 +1269,9 @@ int Wsrep_schema::recover_sr_transactions()
       Wsrep_schema_impl::scan(frag_table, 2, seqno_ll);
       wsrep::seqno seqno(seqno_ll);
 
+      /* This is possible if the server crashes between inserting the
+         fragment into table and updating the fragment seqno after
+         certification. */
       if (seqno.is_undefined())
       {
         Wsrep_schema_impl::delete_row(frag_table);
```

Comment:
At this point the table is opened for read, not sure we are allowed to delete the row.
It is probably sufficient to open the table for write as well.